add code to determine rho

earoy opened the pull request:
TODO: 

- [x] Add Ethan's info here: https://github.com/tractometry/tractable/blob/main/DESCRIPTION#L29
- [x] Vignette that compares with and without autocorr on Sarica data.

arokem commented:
Please add your info here: https://github.com/tractometry/tractable/blob/main/DESCRIPTION#L29

kellychang4 commented:
Had to redefine `formula` inside the current environment for `df_tract` to be recognized. Not an ideal solution to pass test coverage.

arokem commented:
My recent commit tries a slightly different approach, where `as.formula` is called on the string every time it's passed to `bam`.

earoy commented:
I added a vignette comparing a model with and without the AR1 structure. Let me know if there's anything I should flesh out or add...I also added my info to the DESCRIPTION file and updated the urls there to point to `github.com/tractometry/tractable/` instead of the old yeatmanlab links

arokem commented on vignettes/tractable-autocorrelations.Rmd:
```diff
@@ -0,0 +1,108 @@
+
+Examining the summary of the resulting GAM fit object shows us that the inclusion
+of the AR1 model changes the resulting statistics of our model. Although there 
+is still a statistically significant effect of group (p=0.044), the value of the
```
That's quite a change in p-value! Interesting 🤔

earoy replied:
Yeah it's interesting how dramatically things can change based on rho...If it would be useful, I have some code that I can add to this vignette that plots how various statistics (fREML, p-values, k-values, etc...) change based on the value of rho

arokem replied:
I think it would be interesting to maybe show the ACF for this data - it helps understand why the effect is so dramatic. Maybe even use itsadug directly to show that.

arokem commented:
Thanks Ethan -- looks great! I am still working to assuage the CI, but as soon as we can get it to pass, I think we can merge this.

arokem commented:
It passes! 🎉 

I am going to go ahead and merge this now and we can come back to extend the vignette on a separate PR.

earoy commented:
Yay! Sounds like a plan!
